javascript/javascript1/week2/fullName.js

```diff
@@ -0,0 +1,30 @@
+//FLIGHT BOOKING FULLNAME FUNCTIONALITY
+
+console.log('\n***** Homework Javascript Week02 Task 01-FULL NAME" *****');
+
+
+function getFullname(firstname, surname, useFormalName,gender) {
+    if (firstname === '' && surname === '') {
+            return 'Please enter both First Name and Last Name';
+    } else if (useFormalName) {
+        if(gender=='male'){
+                return '\n"' + 'Lord ' + firstname + ' ' + surname + '"';
+            } else {
+                return '\n"' + 'Lady ' + firstname + ' ' + surname + '"';
```

Comment:
When you end a `console.log` on variable, you don't need to close with the quotemarks, unless you really want to add them.
```suggestion
                return 'Lady ' + firstname + ' ' + surname;
```

Reply:
DONE